.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push]
+
+jobs:
+  bench:
+    name: Building and test bench on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, ubuntu-18.04, ubuntu-20.04]
+        include:
+          - target: ["run_all", "run_all_sancus"]
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: sudo apt-get install build-essential cmake iverilog tk binutils-msp430 gcc-msp430 msp430-libc msp430mcu expect-dev verilator -y
+      - name: Build sancus-core
+        run: mkdir build && cd build && cmake .. && cd ..
```

Comment:
It seems I forgot the actual `make` step right? Shouldn't it be cmake .. && make all && cd .. ?

Reply:
no in sancus-core only cmake to set the configurable parameters in verilog, no make needed normally